### PR TITLE
minor formatting

### PR DESCRIPTION
**Pull request by whyboris**
Not very important, but makes my `VS Code` not complain 😅

### CODE REVIEW

**Comment by whyboris** [comment]
Pro tip: add `?w=1` to the URL when reviewing to ignore whitespace changes 😉 
e.g. https://github.com/whyboris/Video-Hub-App/pull/18/files?w=1

File: main-support.ts [hunk 1]
```diff
@@ -231,33 +231,36 @@ export function takeTenScreenshots(
...
-
-  const ffmpeg = spawn(ffmpegPath, args);
-  ffmpeg.stdout.on('data', function (data) {
-    console.log(data);
-  });
-  ffmpeg.stderr.on('data', function (data) {
-    console.log('grep stderr: ' + data);
-  });
-  ffmpeg.on('exit', () => {
+  args.push('-frames', 1, '-filter_complex',
```
Comment (cal2195) [inline main-support.ts]: Can you move `-filter_complex` to the line below, so the flag and params are together?

Comment (whyboris) [reply main-support.ts]: Hope this is what you meant -- please see the below commit

**Comment by whyboris** [comment]
I highly recommend VS Code, _especially_ when working with _Angular_ 👍

**Pull request merged**